core/src/main/java/org/frankframework/configuration/ConfigurationUtils.java

```diff
@@ -160,10 +159,11 @@ public static Map<String, Object> getConfigFromDatabase(ApplicationContext appli
 		try {
 			qs.open();
 			try(Connection conn = qs.getConnection()) {
-				if(version == null) {//Return active config
-					String query = "SELECT CONFIG, VERSION, FILENAME, CRE_TYDST, RUSER FROM IBISCONFIG WHERE NAME=? AND ACTIVECONFIG="+(qs.getDbmsSupport().getBooleanValue(true));
+				if (version == null) { // Return active config
+					String query = "SELECT CONFIG, VERSION, FILENAME, CRE_TYDST, RUSER FROM IBISCONFIG WHERE NAME=? AND ACTIVECONFIG=?";
 					try (PreparedStatement stmt = conn.prepareStatement(query)) {
 						stmt.setString(1, name);
+						stmt.setString(2, qs.getDbmsSupport().getBooleanValue(true));
```

Comment:
Als het zo gemakkelijk was, hadden we dit al gedaan. Gaat die goed met alle drivers!? (nee hier zijn geen tests voor).

Reply:
Er zijn ook dingen die makkelijk te wijzigen waren die ook niet gedaan waren, dus dat vind ik niet zo'n plausibele redenering eigenlijk. Ik heb ook niet vernomen dat je hier al specifiek mee bezig was geweest of dat deze niet makkelijk zijn. 
Ik draai deze dan wel terug, want blijkbaar zijn ze te risicovol, helemaal als ze niet afgetest zijn.

Reply:
Geen tests, dat is vervelend. Anders zou ik zeggen, maak ervan: 

`stmt.setBoolean(2, true);`

Ik geloof hier al eens een keer naar gekeken te hebben? Maar ik weet niet waarom ik toen uiteindelijk besloot om niks aan te passen.

Reply:
Ik zet het wel terug dan. Als er tzt meer test-coverage is, kun je dit wel gaan aanpakken.